package.json

```diff
@@ -18,11 +18,16 @@
     "*.js",
     "test"
   ],
+  "nyc": {
+    "require": ["babel-register"],
+    "reporter": ["text", "html"]
+  },
   "scripts": {
-    "test": "istanbul cover --report cobertura _mocha -- --compilers js:babel-register",
-    "pretest": "godaddy-js-style-lint *.js ./test",
-    "prepublish": "npm run build",
-    "build": "babel *.js -d lib"
+    "build": "babel *.js -d lib",
+    "lint": "eslint --fix *.js test",
```

Comment:
Why are you not using the `eslint-godaddy-react` binary that is installed by the `eslint-config-godaddy-react` package here? This way you don't need to have a the .eslint file in the root of the repository.